src/tools/rest-frontend/package.json

```diff
@@ -36,6 +36,7 @@
         "bootstrap": "^3.3.7",
         "connect-modrewrite": "^0.9.0",
         "fs-extra": "^0.30.0",
+        "font-awesome": "^4.7.0",
```

Comment:
Why do we need this line if we also have the fonts included in the repo?

Reply:
Yeah - we don't :) I'll remove it.